cosmo/monitors/data_models.py

```diff
@@ -192,3 +194,104 @@ def get_new_data(self):
         data_results = data_results.replace([np.inf, -np.inf], np.nan).dropna().reset_index(drop=True)
 
         return data_results[~data_results.EXPTYPE.str.contains('ACQ|DARK|FLAT')]
+
+
+class NUVDarkDataModel(BaseDataModel):
+    """Datamodel for NUV Dark files."""
+    files_source = FILES_SOURCE
+    subdir_pattern = '?????'
```

Comment:
Doesn't look like you're using `subdir_pattern` elsewhere, so I think there are a couple of options: 
1. Get rid of this attribute
2. Use `find_files` (which uses `subdir_pattern`)

I'd recommend the 2nd option just because it might make "forward compatibility" easier (i.e. if dark datasets aren't manually identified in this way in the future). I think @dzhuliya has an example of using `find_files` on the fuv dark branch

Reply:
yes i want to use `find_files` but for some reason it wasn't working for me so i was planning to update that. i'll look at dzhuliya's branch